Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml

```diff
@@ -1,5 +1,5 @@
 id: 871ba14c-88ef-48aa-ad38-810f26760ca3
-name: Long lookback for multiple users forwarding Office mail to same destination
+name: UOffice Mail Forwarding
 description: |
   'This query over Office Activity audit data highlights cases where user mail is being forwarded.'
```

Comment:
Can you update the description a bit also.  Probably needs some mention of why this is interesting.

Reply:
Done Now. Thanks.